README.md

```diff
@@ -1,4 +1,4 @@
-# Terratag by env0 [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+# [<img src="ttlogo.png" width="300">](https://terratag.io) [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
```

Comment:
I guess we wanna have https://terratag.io up and running before we merge this?

Reply:
Omry should be setting the DNS records soon. Probably not a huge deal, but it can wait until the site is live.

Reply:
Terratag.io is live, we're good to merge.